grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -114,11 +118,11 @@ public static Value of(BigDecimal value) {
   }
 
   public static Value of(UUID value) {
+    ByteBuffer bytes = ByteBuffer.allocate(16);
+    bytes.putLong(0, value.getMostSignificantBits());
```

Comment:
As per earlier comments, is there a way to verify that byte-ordering won't be problematic here? Or does JDK's `UUID` guarantee Big-endian handling.
I know `ByteBuffer`s byte order may be changed but not sure if that could be used for testing correctness since ordering with `ByteBuffer.allocate()` would default to whatever platform default is.

Reply:
Can we make it explicit, instead of relying on `default`?

Reply:
> Can we make it explicit, instead of relying on default?

From: https://docs.oracle.com/javase/8/docs/api/java/nio/ByteBuffer.html

> The initial order of a byte buffer is always BIG_ENDIAN.